packages/brokeneck-react/src/components/Groups.js

```diff
@@ -112,7 +150,12 @@ export default function Groups() {
             </TableHead>
             <TableBody>
               {data?.groups.data.map(group => (
-                <TableRow key={group[groupFields.id]}>
+                <TableRow
+                  key={group[groupFields.id]}
+                  onClick={() =>
+                    history.push(`${match.url}/${group[groupFields.id]}`)
+                  }
+                >
```

Comment:
That was a nice to have UX addition. I didn't remove the link because that's how the keyboard navigation was captured, but I can have a look at what you suggest.
I can also remove this and the associated styles right now, but I think it feels nicer with it, rather than having to try and click on the name of the entity on the left as it's fiddly.

I'm not sure what the diff link is pointing to.

Reply:
I have nw updated the code and removed the links. Let me know what you think.